website/content/docs/connect/failover/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs
+page_title: Failover configuration overview
+description: Learn about failover strategies and service mesh features you can implement to route traffic if services become unhealthy or unreachable, including sameness groups, prepared queries, and service resolvers.
+---
+
+# Failover overview
+
+Services in your mesh may become unhealthy or unreachable for many reasons, but you can mitigate some of the effects associated with infrastructure issues by configuring Consul to automatically route traffic to and from failover service instances. This topic provides an overview of approaches to the failover configurations you can implement with Consul.
+
+## Service failover strategies in Consul
+
+There are several methods for implementing failover strategies between datacenters in Consul. You can adopt one of the following strategies based on your deployment configuration and network requirements:
+
+- Configure the `Failover` stanza in a service resolver configuration entry to explicitly define which services should failover and the targeting logic they should follow.
+- Make a prepared query for each service that you can use to automate geo-failover.
+- Create a sameness group to identify partitions with identical namespaces and service instance names to establish default failover targets.
+
+The following table compares these strategies in deployments with multiple datacenters to help you determine the best approach for your service:
+
+| Failover Strategy | Supports WAN Federation | Supports Cluster Peering | Multi-Datacenter Failover Strength | Multi-Datacenter Usage Scenario |
+| :---------------: | :---------------------: | :----------------------: | :--------------------------------- | :------------------------------ |
+| `Failover` stanza | &#9989;                 | &#9989;                  | Enables more granular logic for failover targeting | Configuring failover for a single service or service subset, especially for testing or debugging purposes |
+| Prepared query    | &#9989;                 | &#10060;                 | Central policies that can automatically target the nearest datacenter | WAN-federated deployments where a primary datacenter is configured |
+| Sameness groups   | &#10060;                | &#9989;                  | Group size changes without edits to existing member configurations | Cluster peering deployments with consistently named services and namespaces |
```

Comment:
From chatting with @hashi-derek  Prepared queries are not broadcast over cluster peering, but we do we allow peering results to be returned by PQs now. @wilkermichael  can probably give you a definitive answer about this one though. 





[10:43 AM](https://hashicorp.slack.com/archives/D041D2X56FR/p1686332613001699)
From what I remember, the PQ failovers can return results in other partitions and from peers.
[10:43 AM](https://hashicorp.slack.com/archives/D041D2X56FR/p1686332628363089)
But PQs themselves are not replicated over peering for sure.

Reply:
This one actually does 
```suggestion
| Failover Strategy | Supports WAN Federation | Supports Cluster Peering | Multi-Datacenter Failover Strength | Multi-Datacenter Usage Scenario |
| :---------------: | :---------------------: | :----------------------: | :--------------------------------- | :------------------------------ |
| `Failover` stanza | &#9989;                 | &#9989;                  | Enables more granular logic for failover targeting | Configuring failover for a single service or service subset, especially for testing or debugging purposes |
| Prepared query    | &#9989;                 |   &#9989;              | Central policies that can automatically target the nearest datacenter | WAN-federated deployments where a primary datacenter is configured. Note prepared queries are not replicated over Peers. |
| Sameness groups   | &#10060;                | &#9989;                  | Group size changes without edits to existing member configurations | Cluster peering deployments with consistently named services and namespaces |
```